README.md

```diff
@@ -158,10 +158,11 @@ For more info, see [CocoaPods/CocoaPods#5540](https://github.com/CocoaPods/Cocoa
 <details>
 <summary><b>Carthage: choose the right version</b></summary>
 
-The project is set up with `SWIFT_VERSION=3.0` on `master`.
+The project is set up with `SWIFT_VERSION=4.2` on `master`.
 
 This means that the framework on `master` will build using:
 
+* Swift 4.0/4.2 on Xcode 10
 * Swift 3.1 on Xcode 8.3
 * Swift 3.0 on Xcode 8.2
 * Swift 2.2/2.3 on Xcode 7.x.
```

Comment:
I wonder if that's still true?

iirc, that worked because Xcode 7 didn't know about (thus ignored) the `SWIFT_VERSION=3.0` build setting and thus used 2.2, while Xcode 8 took it into account and used it.

But now that Xcode 8, 9 and 10 all know about `SWIFT_VERSION` and use it, I wonder what happens if we try to build a project with `SWIFT_VERSION=4.2` using an earlier Xcode which doesn't know that version of Swift? Does it error? Does it fallback to the latest version of Swift it knows?

As a consequence, wouldn't that change risk that OHHTTPStubs wouldn't be compatible with earlier Xcode versions especially if integrated with Carthage, which uses the Xcode project to build the framework?

Any Carthage user have an idea about that? @ikesyo maybe?

Reply:
I think there are a far larger percentage of developers on Xcode 10 using Swift 4.2 than there are on older version of Xcode like 7, or 8.